observability/how-to/activate-managed-alerts.mdx

```diff
@@ -0,0 +1,39 @@
+---
+meta: How to activate managed alerts
+  title: How to activate managed alerts
+  description: This page shows you how to activate managed alerts
+content:
+  h1: How to activate managed alerts
+  paragraph: This page shows you how to activate managed alerts
+categories:
+  - observability
+dates:
+  validation: 2022-10-13
+  posted: 2022-10-30
+---
+
+This page shows you how to activate [managed alerts](/observability/cockpit/concepts/#managed-alerts) which notifies you of your products' behavior.
+
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
```

Comment:
it says resolved but I can't see the linebreaks, unless it's just my view? Or does the message box markdown work without them?